market-service/src/main/java/rs/edu/raf/banka1/model/Forex.java

```diff
@@ -0,0 +1,25 @@
+package rs.edu.raf.banka1.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class Forex {
+    @Id
+    private String symbol;
+
+    private String baseCurrency;
+    private String quoteCurrency;
+    private double exchangeRate;
```

Comment:
stavi Double

Reply:
Nije više validno, pošto smo odradili refactor koda sa onim nasljeđivanjem.